src/ArcxAnalyticsSdk.ts

```diff
@@ -455,15 +421,16 @@ export class ArcxAnalyticsSdk {
     content?: string
     [key: string]: unknown
   }): void {
-    return this.event(ATTRIBUTION_EVENT, attributes)
+    return this.event(Events.ATTRIBUTION, attributes)
   }
 
-  /** Logs page visit events. Only use this method is `trackPages` is set to `false`. */
-  page(attributes: { url: string }): void {
-    if (!attributes.url) {
-      throw new Error('ArcxAnalyticsSdk::page: url cannot be empty')
-    }
-    return this.event(PAGE_EVENT, attributes)
+  /**
+   * Logs the current page
+   */
+  page(): void {
+    return this.event(Events.PAGE, {
+      referrer: document.referrer,
+    })
   }
```

Comment:
There any `PAGE` event that may have a `document.referrer` will be added to the `event.referrer` property. 

Ideally there should only be one per `storage_session_id` like we talked about yesterday, but let's see if that assumption holds true once data starts coming in.

@tanghyd This is the one-liner that is responsible for the `referrer` headaches 😆

Reply:
That's right